### PR TITLE
Simplify test 1

### PR DESCRIPTION
**Pull request by wind57**

### CODE REVIEW

File: spring-cloud-kubernetes-fabric8-discovery/src/test/java/org/springframework/cloud/kubernetes/fabric8/discovery/KubernetesDiscoveryClientFilterTest.java [hunk 1]
```diff
@@ -28,26 +28,21 @@
 import io.fabric8.kubernetes.client.KubernetesClient;
```
Comment (wind57) [inline spring-cloud-kubernetes-fabric8-discovery/src/test/java/org/springframework/cloud/kubernetes/fabric8/discovery/KubernetesDiscoveryClientFilterTest.java:28]: simplify some tests and align to how all others look like

**Comment by wind57** [comment]
This and a few others that will follow are just nitpicks in some tests.

**Comment by wind57** [comment]
@ryanjbaxter I can merge 3 PRs that are related to test only into a single one, or have 3 separate ones for easier review. what is your take? thank you.

**Comment by ryanjbaxter** [comment]
Whatever is easiest for you

**Comment by wind57** [comment]
I personally would prefer 3... thx and this one is the first than, ready to look at.

**Pull request merged**